### PR TITLE
Add readme file

### PR DESCRIPTION
**Pull request by sebastian-berlin-wmse**
It contains brief descriptions of the scripts in this repo.

Bug: [T234786](https://phabricator.wikimedia.org/T234786)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# WMSE Scripts
+This repository contains miscellaneous scripts used by [Wikimedia Sverige (WMSE)](https://www.wikimedia.se). Most of these scripts are single file. 
+
+* fortnox_huvudbok_processing - *???*
```
Comment (sebastian-berlin-wmse) [inline README.md]: @lokal-profil, could you add a description for this?

Comment (lokal-profil) [reply README.md]: You can take the first line from the Readme inside the folder (and maybe mention that there is a sub-readme for instructions.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+* python_scripts
+  * extension_documentation.py - *generates wikitext strings to use on extension pages on the [MediaWiki-wiki](https://www.mediawiki.org).*
+  * wikimania_program_list.py - *generate wikitext for program list used for Wikimania 2019.*
+* drive_calc_getSheetName.js - *???*
```
Comment (lokal-profil) [inline README.md]: Google script for the _Burnchart_ Calc document. Used to copy tab name to a cell and to calculate personnel costs.

Comment (lokal-profil) [reply README.md]: Sorry that is a separate script where this is included.
"Google script for Calc to create a function which returns the name of the sheet where a given cell resides."

**Comment by sebastian-berlin-wmse** [comment]
I managed to merge this in a way that Github didn't pick up.

**Pull request closed without merging**